available_images.md

```diff
@@ -181,6 +181,7 @@ HuggingFace Inference Containers
 
 SageMaker Training Compiler Containers
 ===============================
+SageMaker Training Compiler is available in a subset of AWS Regions. For more information, see [Supported Frameworks, AWS Regions, Tested Instances, and Tested Models](https://docs.aws.amazon.com/sagemaker/latest/dg/training-compiler-support.html) in the *SageMaker Training Compiler developer guide*.
```

Comment:
@judyheflin Can we add this into the table at https://github.com/aws/deep-learning-containers/blob/421c57d9e35918f296ab730bb40291879caf61bf/available_images.md?plain=1#L18 as a column?

Reply:
Added SM training compiler column and removed this line. I also standardized the spacing for each graph, which doesn't show up when the markdown is displayed, it just makes it easier to read while editing.